forge-gui/res/cardsfolder/s/swords_to_plowshares.txt

```diff
@@ -1,8 +1,7 @@
 Name:Swords to Plowshares
 ManaCost:W
 Types:Instant
-A:SP$ ChangeZone | Cost$ W | ValidTgts$ Creature | TgtPrompt$ Select target creature | Origin$ Battlefield | Destination$ Exile | RememberLKI$ True | SpellDescription$ Exile target creature. Its controller gains life equal to its power. | StackDescription$ SpellDescription | SubAbility$ DBGainLife
-SVar:DBGainLife:DB$ GainLife | Defined$ RememberedController | LifeAmount$ X | SubAbility$ DBCleanup
-SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
-SVar:X:RememberedLKI$CardPower
+A:SP$ ChangeZone | ValidTgts$ Creature | Origin$ Battlefield | Destination$ Exile | SubAbility$ DBGainLife | SpellDescription$ Exile target creature.
+SVar:DBGainLife:DB$ GainLife | Defined$ TargetedController | LifeAmount$ X | XDesc$ its power | SpellDescription$ Its controller gains life equal to its power.
```

Comment:
Currently all these cards need Remembered or it will break if you use it on a creature with a static controller change